tools/blocktime/readme.md

```diff
@@ -2,7 +2,7 @@
 
 `blocktime` is a simple tool to analyze block production rates of a chain. It scrapes the latest headers through the RPC endpoint of a provided node and calculates the average, min, max and standard deviation of the intervals between the last `n` blocks (default: 100).
 
-To read up on starting a node and exposing the RPC endpoint go to the docs [here](https://docs.celestia.org/nodes/consensus-node)
```

Comment:
[To the reviewers]: According to this [PR](https://github.com/celestiaorg/docs/pull/1472/files), it looks like the new link should be https://docs.celestia.org/nodes/full-consensus-node

Reply:
This part is redundant as there is another [PR](https://github.com/celestiaorg/celestia-app/pull/3229#pullrequestreview-1972261859) fixing it.